src/common/io/read.h

```diff
@@ -29,6 +29,9 @@ bool ioReadOpen(IoRead *this);
 // Read data from IO and process filters
 size_t ioRead(IoRead *this, Buffer *buffer);
 
+// Same as ioRead() but optimized for small reads
```

Comment:
What constitutes a "small read"? As a user, it might be nice to know it is optimized, but I'd, more importantly, want to know how/when to use ioReadSmall vs isRead.

Reply:
That's hard to say for sure -- really depends on the situation. We have lots of "small" reads right now when only a small amount of data is available.

So I guess it's when your intention is make many repetitive reads that are smaller than ioBufferSize().

Reply:
Maybe a comment stating just that would be good.

Reply:
Clarified in commit 0ea578426aab7a582cb8e1bf2b4cfda5205b0d94